go/pkg/yolosvc/driver_github.go

```diff
@@ -18,6 +18,7 @@ type GithubWorkerOpts struct {
 	LoopAfter  time.Duration
 	ClearCache *abool.AtomicBool
 	Once       bool
+	Repos      string
```

Comment:
```suggestion
	ReposFilter      string
```

Actually, this is not a list of repo to watch, but a list of repos that are allowed to watch in the list of repos automatically fetched